docs/modules/user-guide/partials/proc_defining-kubernetes-resources.adoc

```diff
@@ -21,17 +21,17 @@ Complex deployments can be described using Kubernetes or OpenShift resource list
 ----
 schemaVersion: 2.0.0
 metadata:
-  name: MyDevfile
+  name: mydevfile
```

Comment:
This doc has a lot of data overlap with the file proc_adding-kubernetes-component-to-a-devfile.adoc 

I prefer this over proc_adding-kubernetes-component-to-a-devfile.adoc and we should consider removing the other one altogether

Reply:
Great idea, Maysun, and my thoughts as well. Elson, do you have a preference or opinion?

Reply:
I may be missing something but I don't see a lot of overlap info on the two. I think  proc_adding-kubernetes-component-to-a-devfile.adoc provides a couple of concrete examples on the use case in defining a kubernetes resource. Having said that, both of them eventually want to tell users how to define k8s resources. It probably makes sense to merge the two. Feel free to do it in a different PR for the merging if you find this one is ready to go.